MockerTests/MockerTests.swift

```diff
@@ -63,11 +63,11 @@ final class MockerTests: XCTestCase {
         ]).register()
         
         URLSession.shared.dataTask(with: originalURL!) { (data, _, error) in
-            XCTAssert(error == nil)
+            XCTAssertNil(error)
             let image: UIImage = UIImage(data: data!)!
             let sampleImage: UIImage = UIImage(contentsOfFile: MockedData.botAvatarImageFileUrl.path)!
             
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: MockerTests/MockerTests.swift;
  Line: 69;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

Reply:
Not seeing this one locally :(